docs/user/accessing-dashboard/1.6.x-and-below.md

```diff
@@ -1,6 +1,6 @@
 # Accessing Dashboard 1.6.x and below
 
-___TO BE ADDED___
+___It will be moved soon, for now you can [visit Wiki](https://github.com/kubernetes/dashboard/wiki/Accessing-Dashboard---1.7.X-and-above).___
```

Comment:
The links `1.6.X-and-below` and` 1.7.X-and-above` are swapped.

Reply:
BTW, do we need to support 1.6.x and below?

Reply:
For now, it can stay, in the future, we can remove.